jdk9/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# Dockerizing a base images with:
+#
+#   - Ubuntu 14.04
+#   - Oracle JDK 9
+#
+# Build:    docker build -t exoplatform/base-jdk:jdk9 .
+#
+# Run:      docker run -ti exoplatform/base-jdk:jdk9 bash
+
+FROM       phusion/baseimage:0.9.18
+MAINTAINER eXo Platform <docker@exoplatform.com>
+
+ENV TERM xterm
+
+ENV JDK_VERSION_NUMBER 9
+ENV JDK_VERSION_STRING 181
+
+ENV JDK_VERSION ${JDK_VERSION_NUMBER}
+ENV JDK_DOWNLOAD http://download.oracle.com/otn-pub/java/jdk/${JDK_VERSION_NUMBER}+${JDK_VERSION_STRING}/jdk-${JDK_VERSION}_linux-x64_bin.tar.gz
+
+# Cleanup the base image from useless things
+RUN rm -rf /etc/service/sshd/
+
+# Install some useful or needed tools
+ENV DEBIAN_FRONTEND noninteractive
+# --force-confold: do not modify the current configuration file, the new version is installed with a .dpkg-dist suffix. With this option alone, even configuration
+#   files that you have not modified are left untouched. You need to combine it with
+# --force-confdef to let dpkg overwrite configuration files that you have not modified.
+ENV _APT_OPTIONS -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"
+RUN apt-get -qq update \
+  && apt-get -qq -y upgrade ${_APT_OPTIONS} \
+  && apt-get -qq -y install ${_APT_OPTIONS} \
+    wget \
+    curl \
+    expect \
+    htop \
+    unzip \
+  && apt-get -qq -y autoremove \
+  && apt-get -qq -y clean \
+  && rm -rf /var/lib/apt/lists/*
+
+# Create needed directories
+ENV DOWNLOAD_DIR /srv/downloads
+RUN mkdir -p "${DOWNLOAD_DIR}"
+
+# Add some aliases
+RUN echo "alias ll='ls -al --color'" > /etc/profile.d/aliases.sh
+
+# Configure htop for root user
+RUN mkdir -p /root/.config/htop/ && chmod -R 700 /root/.config/htop/
+ADD conf/htoprc.conf /root/.config/htop/htoprc
+
+# Install Oracle Java 9 SDK
+ENV JVM_DIR /usr/lib/jvm
+RUN mkdir -p "${JVM_DIR}"
+
+RUN wget --no-cookies --no-check-certificate \
```

Comment:
please add the -q parameter to limit the useless log lines during the build (like in other Dockerfile): 

> RUN wget --no-cookies --no-check-certificate \

replaced by

> RUN wget -q --no-cookies --no-check-certificate \

Reply:
Done